builtin/logical/ssh/backend_test.go

```diff
@@ -2820,3 +2729,774 @@ func TestProperAuthing(t *testing.T) {
 		t.Fatal("Expected to have validated at least one path.")
 	}
 }
+
+func submitCAIssuerStep(issuerName string, parameters map[string]interface{}) logicaltest.TestStep {
+	path := "issuers/import"
+	if issuerName != "" {
+		path += "/" + issuerName
+	}
+	return logicaltest.TestStep{
+		Operation: logical.UpdateOperation,
+		Path:      path,
+		Data:      parameters,
+	}
+}
+
+func updateIssuersConfigStep(parameters map[string]interface{}) logicaltest.TestStep {
+	return logicaltest.TestStep{
+		Operation: logical.UpdateOperation,
+		Path:      "config/issuers",
+		Data:      parameters,
+	}
+}
+
+// TestSSHBackend_IssuerLifecycle tests the complete lifecycle of issuers including
+// creation, configuration, updates, and deletion
+func TestSSHBackend_IssuerLifecycle(t *testing.T) {
```

Comment:
I'd mark this `t.Parallel()` and maybe consider doing the same for the subtests.